src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -273,6 +273,16 @@ class CreatePipelineRun extends React.Component {
           params: initialParamsState(pipelineInfo[PARAM_SPECS])
         };
       });
+    } else {
+      // Reset pipelineresources and params when no Pipeline is selected
+      this.setState({
+        ...initialPipelineInfoState(),
+        [NAMESPACE]: this.getPipelineInfo('namespace'),
+        [PIPELINE_REF]: '',
+        [RESOURCE_SPECS]: [],
+        [PARAM_SPECS]: [],
+        [PIPELINE_ERROR]: false
```

Comment:
`initialPipelineInfoState` already returns these values, so the only one we need to keep here is `[NAMESPACE]`

Reply:
Have removed these now